### PR TITLE
[chore] Inline google font to fix GDPR issue

### PR DESCRIPTION
**Pull request by janmonschke**
# Description

(fixes https://github.com/usebruno/bruno/issues/695)

- Upgrades to Next 13 in order to use the `next/font/google` module
- Embedds the `Inter` font to make bruno GDPR compliant

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

### CODE REVIEW

File: packages/bruno-app/.babelrc
Comment (j0k3r) [inline packages/bruno-app/.babelrc:1]: Shouldn't `@babel/*` deps be removed then?

Comment (janmonschke) [reply packages/bruno-app/.babelrc:1]: Good point, removed it in https://github.com/usebruno/bruno/pull/696/commits/c1619613b8373bb4689316e64bca071b61819fd9

**Comment by helloanoop** [comment]
@janmonschke There is a v1.0.0 planned on Oct 29th 2023. Upgrading to a major version of Next with days away is a bit risky. 

I welcome and support this PR. Will review this post V1 launch.
Meanwhile, if there is a smaller PR that can introduce the inline google font without upgrading next, please raise a PR for the same.

**Comment by janmonschke** [comment]
Closing in favor of https://github.com/usebruno/bruno/pull/750

**Comment by janmonschke** [comment]
@helloanoop Agreed, let's upgrade to Next 13 after the 1.0 release 👍

**Pull request closed without merging**